include/alpaka/atomic/AtomicOmpCritSec.hpp

```diff
@@ -33,6 +33,9 @@ namespace alpaka
     {
         //#############################################################################
         //! The OpenMP accelerator atomic ops.
+        //
+        //  Atomics can used in the hierarchy level blocks and threads.
```

Comment:
Here and in the other classes the following would be more correct:
`Atomics can be used in the blocks and threads hierarchy levels.`